src/transformers/models/blip/modeling_tf_blip_text.py

```diff
@@ -668,22 +669,22 @@ def get_extended_attention_mask(
     @unpack_inputs
     def call(
         self,
-        input_ids=None,
-        attention_mask=None,
-        position_ids=None,
-        head_mask=None,
-        inputs_embeds=None,
-        encoder_embeds=None,
-        encoder_hidden_states=None,
-        encoder_attention_mask=None,
-        past_key_values=None,
-        use_cache=None,
-        output_attentions=None,
-        output_hidden_states=None,
-        return_dict=None,
-        is_decoder=False,
-        training=None,
-    ):
+        input_ids: TFModelInputType | None = None,
```

Comment:
I've seen a nice mix of this `TFModelInputType` and `tf.Tensor | None` in a few models, so whenever I had to fill `input_ids` I defaulted to the former as  the instructions of the issue say. If this is not the case, is there a way to tell apart when each one should be used?